tests/system/docker-compose.yml

```diff
@@ -99,4 +99,6 @@ services:
       - node-5
       - node-6
       - node-7
-    command: [ "-c", "none", "http://node-7:7826" ]
+    volumes:
+      - "../../ci/:/root/ci/"
```

Comment:
Store it in `tests/system/node/faucet/`

Reply:
Also mount it on `/root/faucet/` and you don't need to provide a command

Reply:
Updated